bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/commands/PipelineRunOptions.java

```diff
@@ -70,4 +70,9 @@ public class PipelineRunOptions extends PipelineOptions {
     @CommandLine.Option(names = { "-pc", "--pipeline-configuration" },
             description = "The Pipeline Configuration File when using the Jenkins Templating Engine")
     public File pipelineConfiguration;
+
+    @CommandLine.Option(names = { "-wlrd", "--write-log-retry-duration" },
+            description = "Initializing build log forwarding to stdout may fail if the build log has not been created yet. " +
+                    "This option sets an overall duration in seconds for retries. A delay between retries is 100ms")
+    public int writeLogRetryDuration = 1;
```

Comment:
The backoff period is not relevant anymore. Even the fact that a retry loop is used doesn't matter.

```suggestion
    @CommandLine.Option(names = { "-wlit", "--write-log-init-timeout" },
            description = "Initializing build log forwarding to stdout may fail if the build log has not been created yet. " +
                    " This option defines the maximum time (in seconds) to wait for build log creation." +
                    " Defaults to 1 second.")
    public int writeLogInitTimeoutSeconds = 1;
```